src/styles/stitches.config.ts

```diff
@@ -54,6 +63,20 @@ export const { styled, css, globalCss, keyframes, getCssText, theme, createTheme
         paddingX: (value: string) => ({ paddingLeft: value, paddingRight: value }),
         paddingY: (value: string) => ({ paddingTop: value, paddingBottom: value }),
         boxSizingForAllChildren: (value: string) => ({ '*': { boxSizing: value } }),
+        elevationBoxShadow: (value: TElevationType) => {
+            if (!Array.isArray(elevation_box_shadow[`${value}`])) {
+                const { x, y, blur, spread, color } = elevation_box_shadow[`${value}`] as TBoxShadowProperties;
```

Comment:
```suggestion
                const { x, y, blur, spread, color } = box_shadow;
```
Once the above check is added, we shouldn't be needing the `as TBoxShadowProperties;`

Reply:
elevation_box_shadow[`${value}`] can have an array or object, as per the number of box-shadows in design-token , so `as TBoxShadowProperties`